pkg/cmd/cli/cmd/build.go

```diff
@@ -196,5 +223,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVar(&(cfg.RuntimeImage), "runtime-image", "", "Image that will be used as the base for the runtime image")
 	buildCmd.Flags().VarP(&(cfg.RuntimeArtifacts), "runtime-artifact", "a", "Specify a file or directory to be copied from the builder to the runtime image")
 	buildCmd.Flags().StringVar(&(networkMode), "network", "", "Specify the default Docker Network name to be used in build process")
+	buildCmd.Flags().StringVarP(&(cfg.AsDockerfile), "as-dockerfile", "", "", "EXPERIMENTAL: Output a Dockerfile to this path instead of building a new image")
```

Comment:
/s/as-dockerfile/to-dockerfile/ (?)

Reply:
i went with `as` because in my mind the default behavior is "build as an image" and the new behavior is "build as a dockerfile artifact", but if you can swing @gabemontero or @coreydaley i'll switch it out.

Reply:
full context for voting:
```
$ s2i build . build-image --as-dockerfile my/Dockerfile
```
vs.
``
$ s2i build . build-image --to-dockerfile my/Dockerfile
``

Reply:
My vote is for --as-dockerfile
$ s2i build . build-image --as-dockerfile my/Dockerfile